### ISSUE TITLE
Support für fireTV v1 (A12GXV8XMS007S)

### ISSUE THREAD

**Issue by SevenUp666** [Apollon77/ioBroker.alexa2#706]
mein fireTV v1 wird zwar von alexa2.1 erkannt, die Objecte enthalten aber keine "commands!`deviceTyp: A2E0SNTXJVT7WK
capabilities: TIMERS_AND_ALARMS,SUPPORTS_LOCALE,SUPPORTS_CONNECTED_HOME_CLOUD_ONLY,MICROPHONE,REMINDERS,SUPPORTS_SOFTWARE_VERSION,SUPPORTS_LOCALE_SWITCH,FLASH_BRIEFING,ASCENDING_ALARM_VOLUME,DIALOG_INTERFACE_VERSION,CUSTOM_ALARM_TONE,SOUND_SETTINGS,ACTIVE_AFTER_FRO,PERSISTENT_CONNECTION,CHANGE_NAME,ARTHUR_TARGET,VOLUME_SETTING`

[HIER](https://github.com/Apollon77/ioBroker.alexa2/issues/19#issuecomment-407205642) steht, dass commandSupport für das Gerät auf false gesetzt wurde. Das erklärt also das aktuelle Verhalten.

Ich habe vor einer Weile eine Datei von alexa2.1 editiert und das Gerät mit commandSupport=true hinzugefügt. Alles lief problemlos. Nach dem Adapter-Update ist meine Änderung wieder weg, und ich erinnere mich nicht mehr, wo es editiert werden soll.

Deutlich besser wäre es natürlich, wenn die Änderung direkt in den Adapter einfließt und dafür schon mal ein Danke!

**Comment by Apollon77** [comment]
Das bedeutet also das Kommandos gehen? Wenn es im Adapter deaktiviert ist gab es einmal eine Rückmeldung das es nicht tut ...

**Comment by SevenUp666** [comment]
Ja, zumind. das Kommando "textCommand" geht bei mir und ich nutze nur dieses.
